LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaPhysical/Rxpk.cs

```diff
@@ -47,7 +47,7 @@ public class Rxpk
         public string Codr { get; set; }
 
         [JsonProperty("rssi")]
-        public int Rssi { get; set; }
+        public double Rssi { get; set; }
```

Comment:
Please revert this change as it's not needed and will be removed anyways with #1153

Reply:
Done in aab1bb53029445d5dfe49891416cf0b458224419